src/test/java/org/jvnet/winp/PlatformSpecificProcessTest.java

```diff
@@ -106,7 +117,42 @@ public void getCommandLine_shouldNotFailIfTheProcessIsDead() throws Exception {
         
         Assert.fail("Expected WinpException since the process is killed");
     }
-    
+
+    @Test
+    public void getEnvironmentVariables_shouldReturnCorrectValues() throws Exception {
+        Process p = spawnTestApp();
+        WinProcess wp = new WinProcess(p);
+        // spawned processes should inherit our environment variables
+        Map<String, String> inheritedEnv = System.getenv();
+        Map<String, String> processEnv = wp.getEnvironmentVariables();
+
+        // environment variable that start with = is just some funky stuff!
+        // and there are some that start with "=" that won't show up in set e.g. =ExitCode =CLINK.SCRIPTS
```

Comment:
I could not find any documentation on this (starting with `=`, but they exist and are funky!
so just excluded them from the test

e.g. 
```batch
set
echo %=ExitCode%
```